src/vs/workbench/contrib/terminal/browser/media/shellIntegration-login.zsh

```diff
@@ -2,8 +2,8 @@
 #   Copyright (c) Microsoft Corporation. All rights reserved.
 #   Licensed under the MIT License. See License.txt in the project root for license information.
 # ---------------------------------------------------------------------------------------------
-if [[ $options[norcs] = off && -o "login" &&  -f $USER_ZDOTDIR/.zlogin ]]; then
-	VSCODE_ZDOTDIR=$ZDOTDIR
-	ZDOTDIR=$USER_ZDOTDIR
-	. $USER_ZDOTDIR/.zlogin
+
+ZDOTDIR=$USER_ZDOTDIR
```

Comment:
always set this here as it is the last file that we run for `login` shells, whether or not the `$ZDOTDIR/.zlogin` exists